hpeOneView/resources/activity/tasks.py

```diff
@@ -38,7 +38,7 @@ class Tasks(ResourcePatchMixin, Resource):
     def __init__(self, connection, data=None):
```

Comment:
Add childLimit and topCount in Document's Args section.

Reply:
Done